etna/core/mixins.py

```diff
@@ -37,6 +38,10 @@ def _get_target_from_class(value: Any):
             return None
         return str(value.__module__) + "." + str(value.__class__.__name__)
 
+    @staticmethod
+    def _get_target_from_function(value: Callable):
+        return str(value.__module__) + "." + str(value.__name__)
```

Comment:
What about `__qualname__`?

Reply:
?
![image](https://user-images.githubusercontent.com/33594071/190370678-41384ac1-2fb4-45b6-8092-19d26705571d.png)

Reply:
`qualname` will behave differently if distance func is a static class method of is sitting somewhere inside class.

https://stackoverflow.com/questions/58108488/what-is-qualname-in-python